convene-web/app/javascript/src/video_room.js

```diff
@@ -21,7 +21,6 @@ export default class VideoRoom {
   }
 
   exitRoom() {
-    this.jitsi.dispose();
```

Comment:
No, going to clean up unnecessary js on next commit.

Reply:
I believe what `dispose()` does is to remove all listeners and remove the iframe.
https://github.com/jitsi/jitsi-meet/blob/342a00a6af323d98eef33fc60f4e2a515c8e254d/modules/API/external/external_api.js#L521-L527
Since we are reloading the page anyway, so I decided to remove all logic using `dispose()`.

Reply:
I am slightly worried it leaves the server thinking the connection is not closed; but we can validate that by seeing that when user A leaves by clicking a button, user B sees them leave immediately.

Reply:
Good concern, I have confirmed once user A leaves, user B sees them leave.
But I wasn't sure how the server was able to detect that.